client/src/pages/Booking/BookingSlot/useStyles.ts

```diff
@@ -0,0 +1,24 @@
+import { Theme } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+export const useStyles = makeStyles((theme: Theme) => ({
+  card: {
+    margin: '10px 0',
```

Comment:
It's preferable to use `theme` integrated spacing for consistent styling throughout the app. Take a look at spacing docs [here](https://mui.com/customization/spacing/) 👍🏼